### DISCUSSION TITLE
智慧後照鏡、電車、油車

### DISCUSSION THREAD

**Discussion by yi-1111** [1101-GS4719-C/Python-Community-GS4719-C#498] [科技新知]
智慧後照鏡最大特色，在於透過數據整合，讓車主能預先得之前方路況，並且以最直覺、簡單方式知曉，進而做到預先的防禦駕駛操作。由於不像四輪車輛可以透過系統輔助做到預先煞車、減速等操作，二輪車輛在安全考量上，許多防禦駕駛還是會由車主手動操作，因此，CROXERA M智慧後照鏡的功用便是協助車主能儘快得知前方路況，並且即時做出正確判斷。

對於喜歡騎車下山的捧友，可以看看這篇文，了解一下未來智慧化機車的趨勢

https://udn.com/news/story/7086/6184133?from=udn-catelistnews_ch2

**Comment by yucheng1028** [comment]
智慧後照鏡恐怕只能讓安全騎士變得更安全，畢竟那些不看或不裝後照鏡的猴子，還是不會去看 : )

**Reply by yi-1111** [reply]
不看或不裝後照鏡的猴子還是別上路了，沒必要搭上人命

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，讓我有初步的認識。

**Comment by sean1325418** [comment]
忽然想到會不會為了看後照鏡的資訊反而沒注意前方路況呢，這樣反而就尷尬了。

**Comment by xkllkx** [comment]
再怎麼智慧，只要仍有三寶，都在劫難逃。

**Comment by ThomasMeng0515** [comment]
這個技術對我來說很新鮮，但是如同上面同學說的，只要有移動神主牌在路上，真的在劫難逃

**Comment by chupochun** [comment]
有些人甚至直接把後照鏡拆掉了，就像是我同學

**Comment by HaoFish123** [comment]
感謝分享，嚷我了解到智慧後照鏡的好處。

**Comment by hapiwangy** [comment]
平時雖然沒再騎機車但會給同學載。這類的科技讓我們離平安上路，安全歸來可以更有保障

**Comment by leon10456** [comment]
如果要再在台灣普及的話 最優先要解決的問題還是駕照太好考 台灣駕駛的安駕觀念過少

**Comment by PoooYen** [comment]
現在的智慧後照鏡已經有盲點偵測跟避免車門開啟了

**Comment by Yuhsin000** [comment]
家裡的車的後照鏡可以偵測後方死角的車，我覺得還不錯用，切換車道時比較安全

**Comment by henrywen547** [comment]
台灣駕照真的太好考，跟實務經驗又沒有太大的關聯
不過這能讓本身就是安全駕駛的人更安全
但不知道三寶用了駕駛習慣會變好
還是繼續認為世界以他們為中心旋轉?

**Comment by gnsJhenJie** [comment]
如果過度依賴智慧後照鏡，或許會讓駕駛的判斷能力下降，裝置故障時駕駛可能就無法做出正確判斷。

**Comment by Lai27330** [comment]
感覺智慧後照鏡增加的功能反而會擾亂騎車的專注度，而且好像沒有從根本解決台灣的交通問題。

**Comment by awkward-willy** [comment]
科技的演進真的能讓人們的生活更好
不過還是得配合知識、道德觀跟社會規範的一併成長才行
否則還是會有很多不願意配合的人，衍伸出更多問題

**Comment by d26488399** [comment]
感謝分享，希望能有效降低車禍發生率

**Comment by stan890425** [comment]
這個之前有被燒到想買~~~
但好貴~~~

**Comment by jcpan821** [comment]
之前還有看過利用光學技術讓A柱透明化，減少視線死角

**Comment by alexthebigpp** [comment]
行車安全如過透過科技來幫忙真的會提升很多

**Comment by Ben6uo** [comment]
但前題還是要把三寶給治一下

**Comment by yenwei12138** [comment]
汽車的電子後照鏡已經蠻常見了，機車的倒是還沒見過

**Comment by etin2468** [comment]
自己再怎麼做好安全防護，但三寶總是能超乎我們的想像阿

**Comment by Lynn12378** [comment]
騎機車要看前方路況同時又要看後照鏡，有了智慧後照鏡感覺可以省不少力氣，也能更加確保行車安全
